src/components/App/App.jsx

```diff
@@ -163,16 +165,10 @@ class App extends React.Component {
         to: '/combos',
```

Comment:
I think this somehow got mixed up in a previous change. Combos and Teams are different paths, teams should go in the main navbar and combos in the drawer.

Also, I think we should add Explorer and API to the main navbar.